src/platform/site-wide/loading-button/LoadingButton.jsx

```diff
@@ -21,8 +22,10 @@ export default function LoadingButton({
     children
   );
 
+  // Switch to va-button-icon once the loading icon is added
   return (
     <button
+      type="submit"
```

Comment:
I don't think this should be added for all uses of this component. In some instances, the `type="submit"` is added by the app: https://github.com/department-of-veterans-affairs/vets-website/blob/4b71fcec7ccb38b72556cc3c5ba11f9425811dd0/src/applications/vaos/components/FormButtons.jsx#L27

In others, it's not treated as a `submit` button: https://github.com/department-of-veterans-affairs/vets-website/blob/2609c646f0ee1316389ef338104780544c71a09b/src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalActionButtons.jsx#L89

Reply:
Good callout - reverted this change